core/src/main/scala/io/estatico/confide/StandardInstances.scala

```diff
@@ -40,13 +40,19 @@ trait StandardInstances {
 
   implicit val configList: FromConf[ConfigList] = FromConf.instance2P(_.getList)
 
-  implicit def iter[A : FromConf]: FromConf[Iterator[A]] = FromConf.instance((c, p) =>
+  implicit def iter[A: FromConf]: FromConf[Iterator[A]] = FromConf.instance((c, p) =>
     c.getList(p).iterator.asScala.zipWithIndex.map { case (v, i) =>
       FromConf.parseValue(c.origin, s"$p[$i]", FromConf[A].get, v.render)
     }
   )
 
-  implicit def list[A : FromConf]: FromConf[List[A]] = iter[A].map(_.toList)
+  implicit def list[A: FromConf]: FromConf[List[A]] = iter[A].map(_.toList)
 
-  implicit def vector[A : FromConf]: FromConf[Vector[A]] = iter[A].map(_.toVector)
+  implicit def vector[A: FromConf]: FromConf[Vector[A]] = iter[A].map(_.toVector)
```

Comment:
I actually prefer the space `A : FromConf` since often you may chain them, e.g. `A : FromConf : ClassTag`